### DISCUSSION TITLE
how can i disable all npcbot gossip by server config?

### DISCUSSION THREAD

**Discussion by zhimingbb** [trickerer/Trinity-Bots#532] [Q&A]
how can i disable all npcbot gossip by server config?

**Comment by trickerer** [comment]
You can't. Currently, anyway.
For static bots gossip menu is an essential feature for player interaction.
For wandering bots gossip is currently enabled for GMs for debugging purposes.

**Comment by zhimingbb** [comment]
i want to disable menu avoid bug cause server down

**Reply by trickerer** [reply]
Why don't you just report it so it gets fixed?

**Reply by zhimingbb** [reply]
i did,but no result

**Reply by zhimingbb** [reply]
can i add your IM?

**Reply by trickerer** [reply]
You can find me in discord
